server/player/handler.go

```diff
@@ -22,6 +22,8 @@ type Handler interface {
 	HandleMove(ctx *event.Context, newPos mgl64.Vec3, newYaw, newPitch float64)
 	// HandleTeleport handles the teleportation of a player. ctx.Cancel() may be called to cancel it.
 	HandleTeleport(ctx *event.Context, pos mgl64.Vec3)
+	// HandleToggleSprint handles when the player starts or stops sprinting
+	HandleToggleSprint(ctx *event.Context, toggle bool)
```

Comment:
The name 'toggle' doesn't make it clear what value this holds. To make it extra clear you could have a `before, after bool`